extensions/mysql/index.js

```diff
@@ -120,7 +124,10 @@ class MySQLExtension extends cli.Extension {
                         return tryCreateUser();
                     }
 
-                    return Promise.reject(error);
+                    return Promise.reject(new cli.errors.CliError({
```

Comment:
Hm this looks redundant, because line 151 puts out a very similar error message. Or is there any difference?

Reply:
This particular catch is taking care of the `CREATE USER` query, whereas the catch in line 151 will take care of everything else that goes wrong. The error produced here will just be passed. But I can remove it, when it is redundant.

Reply:
I wonder if it's worth making the `_query` method handle the error instead and attach something to the error object based on the query string?

Reply:
I'm not sure how I could do this with the `Promise.fromCallback` method? Can you give me a hint? Because I generally think that your suggestion is a very good idea.

Reply:
`Promise.fromCallback` returns a promise, so in the _query method you can just do:
```js
Promise.fromCallback(cb => ...).catch((error) => {
  // do stuff
});
```

Reply:
This is done and pushed... Hope I did it right...